source/guides/database-configuration/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Guides - Database Configuration Overview
+---
+
+# Overview
+
+Before starting your server, you need to configure database link in <code>.env*</code> files.
+
+Open this file for each environment and update <code>DATABASE_URL</code> for your database.
+
+## <a href="http://www.postgresql.org/" target="_blank">PostgreSQL</a>
+
+Setup database variable for the development environment:
+
+```
+# .env.development
+DATABASE_URL="postgres://username:password@localhost/bookshelf_development"
+```
+
+Setup database variable for the test environment:
+
+```
+# .env.test
+DATABASE_URL="postgres://username:password@localhost/bookshelf_test"
+```
+
+## Prepare tables in database
+
+After your database variables setup is done you need to create a table in your database before you're be able to launch a development server.
```

Comment:
you need to create the **database and run the migrations before being able** to launch a development server.